includes/wikia/tests/GlobalTitleTest.php

```diff
@@ -58,16 +58,16 @@ function testNewFromText2() {
 	function testUrlsMainNS() {
 		$this->mockProdEnv();
 
-		$title = GlobalTitle::newFromText( "Timeline", NS_MAIN, 410 ); # yugioh
-		$expectedUrl = "http://yugioh.wikia.com/wiki/Timeline";
+		$title = GlobalTitle::newFromText( "Timeline", NS_MAIN, 147 ); # yugioh
```

Comment:
You might want to change the `# yugioh` comment here and `# wowwiki` on line 69.

Reply:
Got that! Thanks